test/complex.jl

```diff
@@ -351,9 +348,9 @@ end
 
 
 @testset "^ (cpow)" begin
-    #  equivalent to exp(y*log(x))
-    #    except for 0^0?
-    #  conj(x)^conj(y) = conj(x^y)
+    # equivalent to exp(y*log(x))
+    #   except for 0^0?
+    # conj(x)^conj(y) = conj(x^y)
```

Comment:
Perhaps also a candidate for expanding the testset description and removing the comment?